app/src/main/kotlin/com/wire/android/ui/sharing/ImportMediaAuthenticatedViewModel.kt

```diff
@@ -299,6 +345,26 @@ class ImportMediaAuthenticatedViewModel @Inject constructor(
         }
     }
 
+    fun onNewConversationPicked(conversationId: ConversationId) = viewModelScope.launch {
+        importMediaState = importMediaState.copy(
+            selfDeletingTimer = observeSelfDeletionSettingsForConversation(
+                conversationId = conversationId,
+                considerSelfUserSettings = true
+            ).first()
```

Comment:
I'm not convinced to use first here because in meantime when some setting will change we should update it